pkg/config/config.go

```diff
@@ -293,6 +293,7 @@ func initConfig(config Config) {
 	config.BindEnvAndSetDefault("dogstatsd_tags", []string{})
 	config.BindEnvAndSetDefault("dogstatsd_mapper_cache_size", 1000)
 	config.BindEnvAndSetDefault("dogstatsd_string_interner_size", 4096)
+	config.BindEnvAndSetDefault("dogstatsd_entity_id_precedence", false)
```

Comment:
Might be worth adding a comment here or document the related boolean field so it's clear what this new setting accomplishes?

Reply:
Do we need to add this to the datadog.yaml.example ? When will this be set to true ?
If users are meant to use this flag we should document it.

Also if we have a release note describing the new feature related to `dogstatsd_entity_id_precedence` we should edit it to maybe explain this flag to ?

Reply:
Added a release note.